file_sink.go

```diff
@@ -173,11 +173,12 @@ func (fs *FileSink) open() error {
 
 	// Change the file mode in case the log file already existed. We special
 	// case /dev/null since we can't chmod it and bypass if the mode is zero
-	switch fs.Path {
+	switch newfilePath {
 	case "/dev/null":
+	case "/dev/stdout":
```

Comment:
Perhaps special case the handling of stderr as well?

Reply:
Added. Please take a look.